tests/Unit/DeploymentConfiguration/ApplicationConfigurationSnapshotTest.php

```
@@ -0,0 +1,123 @@
+        'start_command' => 'npm run start',
+    ], $attributes));
+}
+
+function markSnapshotTestApplicationDeployed(Application $application): ApplicationDeploymentQueue
+{
+    $deployment = ApplicationDeploymentQueue::create([
+        'application_id' => (string) $application->id,
+        'deployment_uuid' => (string) Str::uuid(),
+        'status' => 'finished',
+        'commit' => 'HEAD',
+    ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Models\ApplicationDeploymentQueue' in method 'markSnapshotTestApplicationDeployed'. (undefined)

(StaticAccess)